x-pack/metricbeat/module/oracle/tablespace/_meta/data.json

```diff
@@ -0,0 +1,43 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "event": {
+        "dataset": "oracle.tablespace",
+        "duration": 115000,
+        "module": "oracle"
+    },
+    "metricset": {
+        "name": "tablespace"
+    },
+    "oracle": {
+        "tablespace": {
+            "data_file": {
+                "id": 19,
+                "name": "/u02/app/oracle/oradata/ORCLCDB/orclpdb1/undotbs01.dbf",
+                "online_status": "ONLINE",
+                "size": {
+                    "bytes": 188743680,
+                    "free": {
+                        "bytes": 187695104
+                    },
+                    "max": {
+                        "bytes": 34359721984
+                    }
+                },
+                "status": "AVAILABLE"
+            },
+            "name": "UNDOTBS1",
+            "space": {
+                "free": {
+                    "bytes": 10092544
+                },
+                "used": {
+                    "bytes": 188743680
+                }
+            }
+        }
+    },
+    "service": {
+        "address": "oracle://sys:Oradoc_db1@localhost:1521/ORCLPDB1.localdomain?connectionClass=\u0026poolIncrement=0\u0026poolMaxSessions=0\u0026poolMinSessions=0\u0026sysdba=1\u0026sysoper=0\u0026sysasm=0\u0026standaloneConnection=0\u0026enableEvents=0\u0026heterogeneousPool=0\u0026prelim=0",
```

Comment:
It'd be nice if only the address to the service appear here and not the rest of parameters, but I guess we have this in other modules.